lab/polymer/index.html

```diff
@@ -5,33 +5,29 @@
   <meta charset="utf-8">
   <title>Palindrom with Polymer demo</title>
   <!-- include web components polyfill -->
-  <script src="../../../webcomponentsjs/webcomponents.js"></script>
-
+  <script src="bower_components/webcomponentsjs/webcomponents.js"></script>
 
   <!-- include Palindrom with dependencies -->
-  <script src="../../../fast-json-patch/src/json-patch-duplex.js"></script>
-  <script src="../../src/palindrom.js"></script>
-  <script src="../../src/palindrom-dom.js"></script>
+  <script src="../../dist/palindrom-dom.js"></script>
 
   <!-- include components for demo -->
-  <link rel="import" href="../../../polymer/polymer.html">
+  <link rel="import" href="bower_components/polymer/polymer.html">
 
   <!-- include polyfills -->
   <script src="polyfills/object-observe.js"></script>
   <script src="polyfills/array-observe.js"></script>
-  <link rel="import" href="../../../dom-bind-notifier/dom-bind-notifier.html">
+  <link rel="import" href="bower_components/dom-bind-notifier/dom-bind-notifier.html">
   <link rel="import" href="components/my-menu-button.html">
-  <link rel="import" href="../../../palindrom-bunny/palindrom-bunny.html">
-
+  <link rel="import" href="bower_components/palindrom-bunny/palindrom-bunny.html">
 
   <link rel="stylesheet" type="text/css" href="css/style.css">
 
   <!-- mock server responses -->
-  <script src="../../test/lib/jasmine-2.2.0/jasmine.js"></script>
+  <script src="../utils/lib/jasmine-2.2.0/jasmine.js"></script>
```

Comment:
We are using jasmine just to make demos work as before?
If so I'd make an issue to get rid of it (replace with axios or smth) in next PR :)